src/main/java/io/vertx/core/http/WebSocketBase.java

```diff
@@ -90,6 +91,15 @@
    */
   String subProtocol();
 
+  /**
+   *  Returns the HTTP response headers during the websocket connection handler.
+   *  <p/>
+   *  After the connection handler has completed the response headers will be null.
+   *
+   * @return the response headers
+   */
+  MultiMap responseHeaders();
```

Comment:
call it `headers()` instead and provide also the request headers on the server, that is surely useful